node-tests/blueprints/dependencyScenarios.js

```diff
@@ -0,0 +1,586 @@
+'use strict';
+
+const bs3Regex = '\\^3\\.\\d+\\.\\d+';
+const bs4Regex = 'next';
+const scenarios = [
+  {
+    dependencies: {
```

Comment:
Perhaps we could name the scenarios to make them more intent-revealing and use that name in the tests.